components/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -141,7 +141,7 @@ class D2LFilterDropdown extends mixinBehaviors([D2L.PolymerBehaviors.FilterDropd
 		this.removeEventListener('d2l-dropdown-close', this._handleDropdownClose);
 	}
 
-	_clearFilters() {
+	clearFilters() {
```

Comment:
I think we should leave the underscore here to indicate that this is supposed to be an internal method.  @jstefaniuk-d2l can pitch in

Reply:
We switched to using this externally here so that the applied filters component can clear the dropdown filters: https://github.com/BrightspaceUI/facet-filter-sort/pull/57/files#diff-53571d5ed7be88c8a70d36ce8872776dR227

Reply:
The alternative would be having the client handle "applied-filters-clear-filters" and "dropdown-clear-filters" separately, which may not be the worst thing in the world, but this seemed slightly more convenient